<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformerFilterDialog</class>
 <widget class="QDialog" name="InformerFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>316</width>
    <height>153</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Фильтр сообщений информатора</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="CPersonComboBox" name="cmbAuthor">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkAuthor">
     <property name="text">
      <string>Автор</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkDate">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkSubject">
     <property name="text">
      <string>Тема</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="edtSubject">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="chkRevisionInfo">
     <property name="text">
      <string>Тип сообщений</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="cmbRevisionInfo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Ревизии</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прочие</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkAuthor</tabstop>
  <tabstop>cmbAuthor</tabstop>
  <tabstop>chkSubject</tabstop>
  <tabstop>edtSubject</tabstop>
  <tabstop>chkDate</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>chkRevisionInfo</tabstop>
  <tabstop>cmbRevisionInfo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InformerFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InformerFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBegDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAuthor</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAuthor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRevisionInfo</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbRevisionInfo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSubject</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtSubject</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
